src/style.css

```diff
@@ -33,6 +33,15 @@ p {
 	line-height: 1.65;
 }
 
+button {
+	background-color: transparent;
+	border: 0;
+	font-family: inherit;
+	font-size: 100%;
+	outline: 0;
```

Comment:
Outline shouldn't be removed because that helps the keyboard users to know what is in focus. I'll add back.